hutch_python/cli.py

```diff
@@ -31,7 +31,12 @@ def setup_cli_env():
     hutch.HAPPI_DB = args.db
 
     # Load objects from the configuration file
-    return load(args.cfg)
+    objs = load(args.cfg)
+
+    # Add cli debug tools
+    objs['_debug_console_level'] = set_console_level
+
```

Comment:
Any thought about putting a `--debug` mode arg. That way you can set up debug logging in the terminal even during device loading? 

```python
if args.debug:
     set_console_level(logging.DEBUG)
objs = load(args.cfg)
```
I know how you feel about command line args ...

Reply:
That's a good idea, will implement